modules/events/events-page/layout/sidebar/total-events/NavItem.tsx

```diff
@@ -7,14 +7,31 @@ import cart from '@/public/images/sidebar_icons/cart.png'
 import newOrder from '@/public/images/sidebar_icons/new-order.png'
 import user from '@/public/images/sidebar_icons/user.png'
 import login from '@/public/images/sidebar_icons/login.png'
-
+import { useAppDispatch, useAppSelector } from '@/modules/redux/app/hooks';
+import camelize from '@/modules/utilities/camelize';
+import { addPreviousFilter, categoriesFilterAdded, categoriesFilterRemoved } from '@/modules/events/events.multifilter.slice';
+import { selectEventTotals } from '@/modules/events/events.slice';
 interface NavItemProps {
     title: string,
     index: number
 }
 
 export default function NavItem({ title, index }: NavItemProps) {
-    const icons = [pageVisit, cart, newOrder, user, login]
+    const dispatch = useAppDispatch()
+    const eventTotals = useAppSelector(selectEventTotals)
+    const previousFilter = useAppSelector(state => state.eventsMultipleFilters.previousFilter)
```

Comment:
You can create a selector in the splice to avoid selecting from state here

Reply:
Isn't that additional lines of code?😂😂